frontend/express/libs/members.js

```diff
@@ -1067,6 +1067,7 @@ membersUtility.findMembers = async function(query = {}) {
 */
 membersUtility.updateMember = async function(query = {}, data = {}, upsert = true) {
     return new Promise((resolve, reject) => {
+        delete data._id; // update on the path '_id' would modify the immutable field '_id'
```

Comment:
Deleting _id would remove this property from the originally passed object too, and the code might rely on having that value.
So instead of simply deleting, we need to do the deep copy and then remove that property from the copy of the object